README.md

```diff
@@ -4,213 +4,7 @@ Servo is a prototype web browser engine written in the
 [Rust](https://github.com/rust-lang/rust) language. It is currently developed on
 64-bit macOS, 64-bit Linux, 64-bit Windows, and Android.
 
-Servo welcomes contribution from everyone.  See
-[`CONTRIBUTING.md`](CONTRIBUTING.md) and [`HACKING_QUICKSTART.md`](docs/HACKING_QUICKSTART.md)
-for help getting started.
+Servo welcomes contribution from everyone.
 
-Visit the [Servo Project page](https://servo.org/) for news and guides.
+Check out [The Servo Book](https://book.servo.org) to get started, or go to [servo.org](https://servo.org/) for news and guides.
```

Comment:
Would it be possible to keep the basic build instructions in the README? It's pretty typical to have a quick getting started section in a project's README. I'm just talking about everything under the "Build Setup" header. I'm happy to move everything else.

Reply:
done in 5f83fd4c550740268ec0bd8d01db41b3a149a50f, but i’ve also incorporated some of the updates made in the book, and added an extra step to actually build servoshell